infra/license/check-result-formatter.ts

```diff
@@ -50,3 +50,10 @@ commentBody.forEach((comment) => {
 resultComment += '</details>';
 
 writeFileSync('license_check_result.md', resultComment);
+
+if (totalWarnCount + totalFailCount === 0) {
+  writeFileSync('license_check_result.pass',"");
+}
+else {
+  writeFileSync('license_check_result.fail',"");
```

Comment:
This file is used in https://github.com/Samsung/ONE-vscode/blob/cde462671a859748ee138d9e857824e3d1fcfcba/.github/workflows/check-daily-license.yml#L68